.gitignore

```diff
@@ -1,4 +1,4 @@
-node_modules
-tmp
 .DS_Store
-
+/node_modules/
```

Comment:
Everybody knows that node_modules is a folder and this is just the gitignore. 
If you remove these changes and if the commented that I made about the grunt test on line 19 I will merge this pull request